src/reducers/comments.spec.tsx

```diff
@@ -372,4 +459,110 @@ describe(__filename, () => {
       expect(dispatch).toHaveBeenCalledWith(errorsActions.addError({ error }));
     });
   });
+
+  describe('selectCommentInfo', () => {
+    it('returns undefined before any data has loaded', () => {
+      expect(
+        selectCommentInfo({ comments: initialState, ...keyParams }),
+      ).toEqual(undefined);
+    });
+
+    it('gets comment info', () => {
+      const versionId = 1;
+      const fileName = 'manifest.json';
+      const line = 321;
+      const state = reducer(
+        undefined,
+        actions.beginComment({ versionId, fileName, line }),
+      );
+
+      expect(
+        selectCommentInfo({ comments: state, versionId, fileName, line }),
+      ).toEqual({
+        beginNewComment: true,
```

Comment:
I think it's important that this test verifies all values because `selectCommentInfo` needs to return everything. Since this selector is used in all other tests, breakage might not be apparent (beyond what the type safety gives us).